app-dicts/wordnet/wordnet-3.0-r6.ebuild

```diff
@@ -0,0 +1,88 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools eutils flag-o-matic
+
+DESCRIPTION="A lexical database for the English language"
+HOMEPAGE="http://wordnet.princeton.edu/"
+SRC_URI="
+	http://wordnetcode.princeton.edu/${PV}/WordNet-${PV}.tar.gz
+	mirror://gentoo/${P}-patchset-1.tar.bz2
+	http://wordnetcode.princeton.edu/wn3.1.dict.tar.gz"
+LICENSE="Princeton"
+
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~sparc-solaris ~x86-solaris"
+IUSE="doc"
+
+# In contrast to what the configure script seems to imply, Tcl/Tk is NOT
+# optional. cf. bug 163478 for details. (Yes, it's about 2.1 but it's
+# still the same here.)
+DEPEND="
+	dev-lang/tcl:0=
+	dev-lang/tk:0="
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/WordNet-${PV}"
+
+PATCHES=(
+	# Don't install into PREFIX/dict but PREFIX/share/wordnet/dict
+	"${WORKDIR}/${P}-dict-location.patch"
+	# Fixes bug 130024, make an additional shared lib
+	"${WORKDIR}/${P}-shared-lib.patch"
+	# Don't install the docs directly into PREFIX/doc but PREFIX/doc/PN
+	"${WORKDIR}/${P}-docs-path.patch"
+	"${WORKDIR}"/${P}-CVE-2008-3908.patch #211491
+	"${WORKDIR}"/${P}-CVE-2008-2149.patch #211491
+
+	"${FILESDIR}"/${P}-tcl8.6.patch
+	"${FILESDIR}"/${P}-format-security.patch
+	"${FILESDIR}"/${P}-src_stubs_c.patch
+	"${FILESDIR}"/${P}-fix-indexing-bug-314799.patch
+)
+
+src_prepare() {
+	epatch "${PATCHES[@]}"
```

Comment:
Yes, there is partial success with -p1. Have problem with applying wordnet-3.0-CVE-2008-2149.patch. That's where the ebuild stops.